test/performance/vectorization/vectorPragmas/cForLoopInParIterForeach.chpl

```diff
@@ -0,0 +1,47 @@
+module iters {
+  // TODO remove this test?
+  iter myiter(nn: int, nt: int) {
+    foreach i in 0..#nt {
+      foreach j in i*nn..#nn {
+        yield j;
+      }
+    }
+  }
+
+  // C for loop in standalone with yield should get vector pragma
+  iter myiter(nn: int, nt: int, param tag: iterKind) where tag == iterKind.standalone {
+    coforall i in 0..#nt {
+      const followThis = i*nn..#nn;
+      var idx: followThis.first.type;
+      const first = followThis.first;
+      const str   = followThis.stride;
+      const end   = followThis.last + str;
+      while __primitive("C for loop",
```

Comment:
I thought that c for loop required the pragma on the iterator? Is the compiler adjusting this one somewhere?

Reply:
This is a just-yield loop, so the compiler marks it without the pragma